mfe/pages/dash/DonutGraph.tsx

```diff
@@ -0,0 +1,61 @@
+import React, { useState, useEffect } from "react";
+import dynamic from "next/dynamic";
+import axios from "axios";
+
+const Chart = dynamic(() => import("react-apexcharts"), { ssr: false });
+
+const PieChart = () => {
+  const [chartData, setChartData] = useState<{ labels: string[]; series: number[] }>({
+    labels: [],
+    series: [],
+  });
+
+  useEffect(() => {
+  
+    const fetchData = async () => {
+      try {
+        const response = await axios.get("/api/transactions/teste"); 
+        const data = response.data;
+
+        // Processar os dados para agrupar por tipo de transação
+        const groupedData = data.reduce((acc: any, item: any) => {
+          const type = item.transactionType || "Outros";
+          acc[type] = (acc[type] || 0) + item.amount;
```

Comment:
Acho que assim o código fica menos obscuro e melhor para ler. Que acha?

```suggestion
          acc[type] ||= 0;
          acc[type] += item.amount;
```